app/cronjob/trailingTrade/step/remove-last-buy-price.js

```diff
@@ -169,15 +171,30 @@ const execute = async (logger, rawData) => {
     return data;
   }
 
-  if (baseAssetQuantity * currentPrice < parseFloat(minNotional)) {
+/**
+ * Get symbol configuration
+ *
+ * @param {*} logger
+ * @param {*} rawData
+ */
+ 
+ //Get symbol config
+  const symbolConfiguration = await getConfiguration(logger, symbol);
+  
+  //Define variable
+    var lastBuyThreshold = symbolConfiguration.buy.lastBuyThreshold;
```

Comment:
Do not use `var`. Use `const`.

And try to use destructing assignment for `lastBuyThreshold`.
Refer: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Reply:
Hi. I didn't quite understand why should we use destructuring assignment. Can you clarify?

Reply:
See this https://crunchtech.medium.com/object-destructuring-best-practice-in-javascript-9c8794699a0d

Reply:
I think I got it. Thanks. See commit in a few minutes.